src/js/setup.js

```diff
@@ -18,7 +18,7 @@ let videojs;
 const autoSetup = function() {
 
   // Protect against breakage in non-browser environments.
-  if (!Dom.isReal()) {
+  if (!Dom.isReal() || !videojs.options.autoSetup) {
```

Comment:
while this would work, it still runs the initial timer and makes the comment above incorrect.

Reply:
ok I will change this